documentation/book/ref-kafka-user.adoc

```diff
@@ -139,16 +139,17 @@ Use `\*` to allow or deny the operation from all hosts.
 The `host` field is optional and when not specified, the value `*` will be used as default.
 
 `resource`:: Specifies the resource for which does the rule apply.
-Simple Authorization supports 3 different resource types:
+Simple Authorization supports 4 different resource types:
 +
 * Topics
 * Consumer Groups
 * Clusters
+* Transactional IDs
 +
 The resource type can be specified in the `type` property.
-Use `topic` for Topics, `group` for Consumer Groups and `cluster` for clusters.
+Use `topic` for Topics, `group` for Consumer Groups, `cluster` for clusters, and `transactionalId` for Transactional IDs.
 +
-Topic and Group resources additionally allow to specify the name of the resource for which the rule applies.
+Topic, Group, and Transactional ID resources additionally allow to specify the name of the resource for which the rule applies.
```

Comment:
_Reword as follows:_

Additionally, Topic, Group, and Transactional ID resources allow you to specify the name of the resource for which the rule applies.